### ISSUE TITLE
CI Linter is failing due to bug in Github

### ISSUE THREAD

**Issue by carygravel** [PhilterPaper/Perl-PDF-Builder#138]
See [this issue](https://github.com/github/super-linter/issues/938)

We can implement the workaround, which would shut up the warnings, subscribe to the above thread, and take away the workaround when they say they have fixed things, or we can just put up with them and ignore them, hoping it will be soon.

**Comment by PhilterPaper** [comment]
Yeah, I saw it last night when I pushed an update to PDF::Table. I figured it was somewhere over in the GitHub Actions universe, and was nothing that I had done, and it would be very noticeable to a _lot_ of people. It appears that the appropriate people are aware of the problem and are working on it, so I'm inclined to do nothing and just wait for them to roll back their latest change that broke the system. If it takes them more than a day or two, I'll have to think about some sort of workaround.
